### ISSUE TITLE
IBMI CODE COVERAGE: TESTS -  List of tests are not displayed after version 1.5.11

### ISSUE THREAD

**Issue by devinhh** [halcyon-tech/vscode-ibmi#886]
Version 1.5.11
![image](https://user-images.githubusercontent.com/35005044/193086721-b943fd04-da43-4312-8bf7-c79d516465d4.png)

Version 1.5.15
![image](https://user-images.githubusercontent.com/35005044/193087150-7d8315bd-d989-49bd-a2ad-8621054a6437.png)

* QCCSID: 37
* Features:
   * git: true
   * grep: true
   * tn5250: true
   * setccsid: true
   * GENCMDXML.PGM: true
   * QZDFMDB2.PGM: true
   * iconv: true
   * attr: true
* SQL enabled: true
* Source dates enabled: true

Variants
```json
{
  "american": "%23@$",
  "local": "%23@$"
}
```

Errors:
```json
[
  {
    "command": "system \"DLTOBJ OBJ(ILEDITOR/O_*) OBJTYPE(*FILE)\"",
    "code": 255,
    "stderr": "CPF2125: No objects deleted.",
    "cwd": "/home"
  },
  {
    "command": "system \"CHKOBJ OBJ(QSYS/QCPTOIMPF) OBJTYPE(*DTAARA)\"",
    "code": 255,
    "stderr": "CPF9801: Object QCPTOIMPF in library QSYS not found.",
    "cwd": "/home"
  },
  {
    "command": "system \"CHKOBJ OBJ(QSYS/QCPFRMIMPF) OBJTYPE(*DTAARA)\"",
    "code": 255,
    "stderr": "CPF9801: Object QCPFRMIMPF in library QSYS not found.",
    "cwd": "/home"
  }
]
```

**Comment by worksofliam** [comment]
Coming from Code Coverage

<img width="507" alt="image" src="https://user-images.githubusercontent.com/3708366/193089567-dd176256-4bda-4736-92e2-9836de9a6598.png">

**Comment by worksofliam** [comment]
@devinhh Sorry about that. This was regards to a large internal change that happened and Code Coverage was still using one of the old APIs that was removed.

The Code Coverage extension has just been published an update to 0.1.6 and should be available in 5/10 minutes inside of VS Code. Update and let me know if the problem is fixed. Thanks!

**Comment by devinhh** [comment]
Yes, that fixed this problem.
